src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotConstant.java

```diff
@@ -29,9 +29,29 @@
  */
```

Comment:
Let's take this opportunity to improve this javadoc:
```
/**
 * A value in a space managed by Hotspot (e.g. heap or metaspace).
 * Some of these values can be referenced with a compressed pointer (32 bits)
 * instead of a full word-sized pointer.
 */
```

Reply:
Thanks for the note! By adjustable you mean it can go beyond 32 bits?

Reply:
No, it will be smaller. Lilliput 1 will probably ship with 22 bit narrowKlass, and we may reduce this further.